### ISSUE TITLE
Problem with mouse events over pg plots in taurus guis

### ISSUE THREAD

**Issue by cmft** [taurus-org/taurus#905]
The mouse events (right click) in the pyqtgraph.PlotWidget seem do not work when the PlotWidget is part of a TaurusGui.

You can reproduce the bug with this snippet:

```
import sys
from taurus.external.qt import Qt
import pyqtgraph as pg


def qapp():
    app = Qt.QApplication(sys.argv)
    mainWin = Qt.QMainWindow()
    plot = pg.PlotWidget()
    dock = Qt.QDockWidget('Plot', None)
    dock.setWidget(plot)
    mainWin.addDockWidget(Qt.Qt.TopDockWidgetArea, dock)
    mainWin.show()
    sys.exit( app.exec_() )


def tapp():
    from taurus.qt.qtgui.application import TaurusApplication
    from taurus.qt.qtgui.taurusgui import TaurusGui
    app = TaurusApplication()
    mainWin = TaurusGui()
    plot = pg.PlotWidget()
    dock = Qt.QDockWidget('Plot', None)
    dock.setWidget(plot)
    mainWin.addDockWidget(Qt.Qt.TopDockWidgetArea, dock)
    mainWin.show()
    sys.exit( app.exec_() )


if __name__ == "__main__":
    # qapp() # Works
    tapp()   # Does not work
```

**Comment by cmft** [comment]
The taurusgui example can be simplified changing
```
    dock = Qt.QDockWidget('Plot', None)
    dock.setWidget(plot)
    mainWin.addDockWidget(Qt.Qt.TopDockWidgetArea, dock)
```

by

```
    mainWin.createPanel(plot, 'plot')
```

**Comment by cmft** [comment]
The problem can also be reproduced using `TaurusMainWindow` instead of `TaurusGui`

**Comment by cmft** [comment]
The problem is in `TaurusMainWindow.__init__` method:     

```
        if self._showTaurusMenu:  # Taurus Menu
            self.createTaurusMenu()
```

**Comment by cpascual** [comment]
#906 should fix this (the following snippet did not allow to show the context menu before and is fixed with #906)

```python
import sys
from taurus.qt.qtgui.application import TaurusApplication
from taurus.qt.qtgui.container import TaurusMainWindow
import pyqtgraph as pg

app = TaurusApplication()
mainWin = TaurusMainWindow()
plot = pg.PlotWidget()
mainWin.setCentralWidget(plot)
mainWin.show()
sys.exit( app.exec_() )
```

Thanks to @cmft for pointing out the root of the issue
